docs/ENVIRONMENT.md

```diff
@@ -559,6 +559,12 @@ SYSTEMD_HOME_DEBUG_SUFFIX=foo \
   `mkfs` when formatting LUKS home directories. There's one variable for each
   of the supported file systems for the LUKS home directory backend.
 
+* `$SYSTEMD_HOME_LOCK_SKIP_FREEZE_SESSION` - Takes a boolean. When true, the user's
```

Comment:
as with the other env var: please invert this, and default to true...

i am not a big fan of "negative knobs", if they are easy to avoid.